src/components/variable.jsx

```diff
@@ -232,14 +243,71 @@ const Variable = React.createClass({
+
+  splitAndLinkParam(text) {
+    const paramVariable = []
+    const splits = text.split(/([\S]*[\s]*[=][\s]* parameters\([\S]*\)\.[\S]*[,\s])/)
+    // The first pass will go through each substring and find parameters(x=parameter(x).x.x) and check if they are a node or an actual parameter.
+    // If it's a node, it records the node and the variable associated. Else, it returns the link.
+    let firstPass = splits.map((substring) => {
+      if (this.isParameterCall(substring)) {
+        const parameterCall = substring.match(/([\S]*)([\s]*[=][\s]*parameters\([\S]*\)\.)([\S]*)[,\s]/)
+          if (this.isParameterLeaf(substring)){
+            substring = [parameterCall[1], parameterCall[2], this.linkParam(parameterCall[3], parameterCall[3])]  // Concatenate JSX with a string (+ doesn't work).
+          } else {
+            const regleParamAbstrait = new RegExp(`(${parameterCall[1]}\\.[0-9a-zA-Z\\_]*)`)
+            const regleParamAbstraitMatch = new RegExp(`(${parameterCall[1]}\\.([0-9a-zA-Z\\_]*))`)
+            paramVariable.push({
+              'letter': parameterCall[1],
+              'abstraction': parameterCall[3],
+              'regexSplit': regleParamAbstrait,
+              'regexMatch': regleParamAbstraitMatch})
+            substring = [substring]
+          }
+      } else {
+        substring
+      }
+      return substring
+    })
+    // this takes all the constructed variables and checks the substring to look for parameters
+    paramVariable.map((element)=> {
+      firstPass = firstPass.map((substring) => {
+        if (typeof substring == 'object'){
+          return substring
+        }
+        return substring
+          .split(element.regexSplit)
+          .map((substring2) => {
+            const match = substring2.match(element.regexMatch)
+            if (match){
+              const linkThisParam = this.linkParam(`${element.abstraction}.${match[2]}`, match[2])
+              substring2 = [element.letter, '.', linkThisParam]
+            }
+            return substring2
+          })
+      })
+
+      return element
+    })
+    firstPass = [].concat.apply([], firstPass)
+    return firstPass
+  },
   // Change every OpenFisca variable in the formula by a link to the variable page:
   renderLinkedFormulaVariables(formula) {
     // Split on double quotes first (preventing collision with Link):
-    const splits = this.splitAndLink(formula, '"')
-    return splits.map((substring) => {
+    let splits = this.splitAndLinkParam(formula)
+
+    splits = splits.map((substring) => {
       // Only split strings, as trying to split JSX Links would raise an error
       return typeof substring == 'string' ? this.splitAndLink(substring, '\'') : substring
     })
+    splits = [].concat.apply([], splits)
+    splits = splits.map((substring) => {
+      // Only split strings, as trying to split JSX Links would raise an error
+      return typeof substring == 'string' ? this.splitAndLink(substring, '\"') : substring
```

Comment:
I'd move this type checking and the flattening inside the `splitAndLink ` function to avoid repetition and simplify this one.

Also, if you import `is` from `ramda`, you can use `is(String, substring)` 🎉